src/View/Form/EntityContext.php

```diff
@@ -230,6 +248,26 @@ public function val($field)
     }
 
     /**
+     * Get default value from table schema for given entity field.
+     *
+     * @param string $field Field name.
+     * @param \Cake\Datasource\EntityInterface $entity The entity.
+     * @return mixed
+     */
+    protected function _schemaDefault($field, $entity)
+    {
+        $table = $this->_getTable($entity);
+        if ($table === false) {
+            return null;
+        }
+        $defaults = $table->schema()->defaultValues();
```

Comment:
Should we add a method to get the default value for a specific column? `schema()->defaultValue($field)`?